conformance/utils/config/listener.go

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+// DynamicPortRange specifies the starting and ending port of a
+// range
+type DynamicPortRange struct {
+	Start int
+	End   int
+}
+
+// ListenerConfig allow conformance test runners to configure
+type ListenerConfig struct {
+	DynamicPortRange DynamicPortRange
```

Comment:
What do you think about setting this field inline to simplify customization via options?

Reply:
I think I prefer the verbosity since there could be another `*PortRange` type for some other purpose.